wagmi.config.ts

```diff
@@ -1,2029 +1,1388 @@
-      "name": "getClaimableAmountForOperator",
-      "outputs": [
-          {
-              "internalType": "uint256",
-              "name": "",
-              "type": "uint256"
-          }
-      ],
-      "stateMutability": "view",
-      "type": "function"
-  },
-  {
-      "inputs": [],
-      "name": "getPeriodDistributionAmount",
-      "outputs": [
-          {
-              "internalType": "uint256",
-              "name": "",
-              "type": "uint256"
-          }
-      ],
-      "stateMutability": "view",
-      "type": "function"
+    inputs: [],
+    name: 'totalSupply',
+    outputs: [{ internalType: 'uint256', name: '', type: 'uint256' }],
+    stateMutability: 'view',
+    type: 'function',
   },
   {
-      "inputs": [],
-      "name": "mainnetClaim",
-      "outputs": [],
-      "stateMutability": "nonpayable",
-      "type": "function"
+    inputs: [
+      { internalType: 'address', name: 'to', type: 'address' },
+      { internalType: 'uint256', name: 'value', type: 'uint256' },
+    ],
+    name: 'transfer',
+    outputs: [{ internalType: 'bool', name: '', type: 'bool' }],
+    stateMutability: 'nonpayable',
+    type: 'function',
   },
   {
-      "inputs": [],
-      "name": "operatorClaim",
-      "outputs": [],
-      "stateMutability": "nonpayable",
-      "type": "function"
+    inputs: [
+      { internalType: 'address', name: 'from', type: 'address' },
+      { internalType: 'address', name: 'to', type: 'address' },
+      { internalType: 'uint256', name: 'value', type: 'uint256' },
+    ],
+    name: 'transferFrom',
+    outputs: [{ internalType: 'bool', name: '', type: 'bool' }],
+    stateMutability: 'nonpayable',
+    type: 'function',
   },
   {
-      "inputs": [
-          {
-              "internalType": "uint256",
-              "name": "length",
-              "type": "uint256"
-          }
-      ],
-      "name": "setActivePeriodLength",
-      "outputs": [],
-      "stateMutability": "nonpayable",
-      "type": "function"
-  },
-  {
-      "inputs": [
-          {
-              "internalType": "uint256",
-              "name": "amount",
-              "type": "uint256"
-          }
-      ],
-      "name": "setPeriodDistributionAmount",
-      "outputs": [],
-      "stateMutability": "nonpayable",
-      "type": "function"
-  }
+    inputs: [{ internalType: 'address', name: 'newOwner', type: 'address' }],
+    name: 'transferOwnership',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    inputs: [],
+    name: 'version',
+    outputs: [{ internalType: 'string', name: '', type: 'string' }],
+    stateMutability: 'view',
+    type: 'function',
+  },
 ] as const
 
+const mainnetRiverTokenAddress = '0x53319181e003E7f86fB79f794649a2aB680Db244' satisfies Address
+const baseRiverTokenAddress = '0x91930fd11ABAa5241241d3B07c02A8d0B5ac1920' satisfies Address
+const baseGammaSepoliaTokenAddress = '0x24e3123E1b30E041E2df26Da9d6140c5B07Fe4F0' satisfies Address
```

Comment:
in the next `@river-build/generated` version, we can replace this with the values from river